egs/wsj/s5/steps/make_index.sh

```diff
@@ -72,14 +71,26 @@ if [ ! -z $silence_word ]; then
   silence_opt="--silence-label=$silence_int"
 fi
 
-$cmd JOB=1:$nj $kwsdir/log/index.JOB.log \
-  lattice-add-penalty --word-ins-penalty=$word_ins_penalty "ark:gzip -cdf $decodedir/lat.JOB.gz|" ark:- \| \
-    lattice-align-words $silence_opt --max-expand=$max_expand $word_boundary $model  ark:- ark:- \| \
-    lattice-scale --acoustic-scale=$acwt --lm-scale=$lmwt ark:- ark:- \| \
-    lattice-to-kws-index --max-states-scale=$max_states_scale --allow-partial=true \
-    --max-silence-frames=$max_silence_frames --strict=$strict ark:$utter_id ark:- ark:- \| \
-    kws-index-union --skip-optimization=$skip_optimization --strict=$strict --max-states=$max_states \
-    ark:- "ark:|gzip -c > $kwsdir/index.JOB.gz" || exit 1
-    
+word_boundary=$langdir/phones/word_boundary.int
+if [ -f $word_boundary ] ; then
+  $cmd JOB=1:$nj $kwsdir/log/index.JOB.log \
+    lattice-add-penalty --word-ins-penalty=$word_ins_penalty "ark:gzip -cdf $decodedir/lat.JOB.gz|" ark:- \| \
+      lattice-align-words $silence_opt --max-expand=$max_expand $word_boundary $model  ark:- ark:- \| \
+      lattice-scale --acoustic-scale=$acwt --lm-scale=$lmwt ark:- ark:- \| \
+      lattice-to-kws-index --max-states-scale=$max_states_scale --allow-partial=true \
+      --max-silence-frames=$max_silence_frames --strict=$strict ark:$utter_id ark:- ark:- \| \
+      kws-index-union --skip-optimization=$skip_optimization --strict=$strict --max-states=$max_states \
+      ark:- "ark:|gzip -c > $kwsdir/index.JOB.gz" || exit 1
+else
+  align_lexicon=$langdir/phones/align_lexicon.int 
```

Comment:
yes, sorry. I will try to finish the fix this week
y.

On Sun, Jul 17, 2016 at 9:25 PM, Daniel Povey <notifications@github.com>
wrote:

> In egs/wsj/s5/steps/make_index.sh
> <https://github.com/kaldi-asr/kaldi/pull/868#discussion_r71094151>:
>
> > -    --max-silence-frames=$max_silence_frames --strict=$strict ark:$utter_id ark:- ark:- \| \
> > -    kws-index-union --skip-optimization=$skip_optimization --strict=$strict --max-states=$max_states \
> > -    ark:- "ark:|gzip -c > $kwsdir/index.JOB.gz" || exit 1
> > -
> > +word_boundary=$langdir/phones/word_boundary.int
> > +if [ -f $word_boundary ] ; then
> > +  $cmd JOB=1:$nj $kwsdir/log/index.JOB.log \
> > +    lattice-add-penalty --word-ins-penalty=$word_ins_penalty "ark:gzip -cdf $decodedir/lat.JOB.gz|" ark:- \| \
> > +      lattice-align-words $silence_opt --max-expand=$max_expand $word_boundary $model  ark:- ark:- \| \
> > +      lattice-scale --acoustic-scale=$acwt --lm-scale=$lmwt ark:- ark:- \| \
> > +      lattice-to-kws-index --max-states-scale=$max_states_scale --allow-partial=true \
> > +      --max-silence-frames=$max_silence_frames --strict=$strict ark:$utter_id ark:- ark:- \| \
> > +      kws-index-union --skip-optimization=$skip_optimization --strict=$strict --max-states=$max_states \
> > +      ark:- "ark:|gzip -c > $kwsdir/index.JOB.gz" || exit 1
> > +else
> > +  align_lexicon=$langdir/phones/align_lexicon.int
>
> @jtrmal <https://github.com/jtrmal>, I think you might have forgotten
> about this.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/868/files/1965bb71d1637d4189d57b50ea4a38331ec7d1c7#r71094151>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKisX1qqp8hHfd3_GAfNTp9963qgnUb9ks5qWtYHgaJpZM4I_PMB>
> .
>